src/utilities.js

```diff
@@ -29,14 +29,11 @@ export function assert(condition, message) {
 
 // Remove code modifications.
 export function cleanupSource(src) {
-  /* START_TESTS_ONLY */
-
   // Restore eval which is modified by esm module.
   src = src.replace(/\(0,[^)]+\)/g, '(0, eval)');
```

Comment:
This is getting really close!

Can you please focus the regexp a little more rather than just relying on `(0,any(thing), after a zero comma)` to mean `(0, eval)`?  I'd suggest changing the calls to `('indirectEval', eval)` and then matching on that pattern.

Reply:
How about we implement the fix from the "other" branch?

```js
export function cleanupSource(src) {

  // Restore eval which is modified by esm module.
  // (0, eval) => (0, _<something>.e)
  src = src.replace(/\(0, _[^.]+\.e\)/g, '(0, eval)');

  // Restore Reflect which is modified by esm module.
  // Reflect => _<something>.e.Reflect
  src = src.replace(/_[^.]+\.g\.Reflect/g, 'Reflect');

  // Remove code coverage which is injected by nyc module.
  src = src.replace(/cov_[^+]+\+\+[;,]/g, '');

  return src;
}